src/native/fs/fs_napi.cpp

```diff
@@ -45,6 +46,7 @@ const static std::map<std::string, int> flags_to_case = {
     { "rs+", O_RDWR | O_SYNC },
     { "sr+", O_RDWR | O_SYNC },
     { "w", O_TRUNC | O_CREAT | O_WRONLY },
+    { "wt", O_RDWR | __O_TMPFILE },
```

Comment:
why are we using `__O_TMPFILE` and not `O_TMPFILE`?
If there's a good reason, then we better explain that in a comment here.

Reply:
fixed